Add refresh, flush, clear cache, and notifications documentation for index management in Dashboards (#4207)
opensearch-project/documentation-website: _dashboards/im-dashboards/index-management.md

<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/split-expanded.png" alt="User interface showing split page">

### Refreshing an index

Refreshing an index makes new updates to the index visible to search operations. 

The refresh operation can be applied only to open indexes. 

> To refresh all indexes, from the **Actions** dropdown list, select **Refresh**.

Reviewer: ```suggestion
To refresh all indexes, select **Refresh** from the **Actions** dropdown list.
```

> To refresh a particular index, select the index from the **Indices** list under **Index Management**. Then from the **Actions** dropdown list, select **Refresh**. 

Reviewer: ```suggestion
To refresh a particular index, select the index from the **Indices** list under **Index Management**. Then select **Refresh** from the **Actions** dropdown list. 
```

### Flushing an index

The flush operation performs a Lucene commit, writing segments to disk and starting a new translog. 

The flush operation can be applied only to open indexes. 

To flush all indexes, from the **Actions** dropdown list, select **Flush**.

> To flush a particular index, select the index from the **Indices** list under **Index Management**. Then from the **Actions** dropdown list, select **Flush**. 

Reviewer: ```suggestion
To flush a particular index, select the index from the **Indices** list under **Index Management**. Then select **Flush** from the **Actions** dropdown list. 
```

### Clearing an index's cache

The [clear cache operation]({{site.url}}{{site.baseurl}}/api-reference/index-apis/clear-index-cache/) can be applied only to open indexes. 

> To clear cache for all indexes, from the **Actions** dropdown list, select **Clear cache**.

Reviewer: ```suggestion
To clear all index caches, select **Clear cache** from the **Actions** dropdown list.
```

> <img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/create-alias.PNG" alt="User interface showing creat Alias page">

Reviewer 2: ```suggestion
<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/create-alias.PNG" alt="User interface showing create Alias page">
```

<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/create-alias.PNG" alt="User interface showing creat Alias page">

### Editing an alias

To edit an alias, perform the following steps:

1. Select the alias you want to edit.
2. Choose the **Actions** button.
3. Choose **Edit** from the dropdown list.

### Deleting an alias

To delete an alias, perform the following steps:

1. Select the alias you want to edit.
2. Choose the **Actions** button.
3. Choose **Delete** from the dropdown list.

### Refreshing an alias

Refreshing an alias makes new updates to the index visible to search operations. 

The refresh operation can be applied only to open indexes associated with the specified aliases.

To refresh a particular alias, select the alias from the **Aliases** list under **Index Management**. Then from the **Actions** dropdown list, select **Refresh**. 

### Flushing an alias

The flush operation performs a Lucene commit, writing segments to disk and starting a new translog. 

The flush operation can be applied only to open indexes associated with the specified aliases.

> To flush all aliases, from the **Actions** dropdown list, select **Flush**.

Reviewer 3: Please remove this line, flush all only applies to index, not alias

To flush a particular alias, select the alias from the **Aliases** list under **Index Management**. Then from the **Actions** dropdown list, select **Flush**. 

### Clearing an alias's cache

The [clear cache operation]({{site.url}}{{site.baseurl}}/api-reference/index-apis/clear-index-cache/) can be applied only to open indexes associated with the specified aliases.

> To clear cache for all open aliases, from the **Actions** dropdown list, select **Clear cache**.

Reviewer 3: Please remove this line, clear cache for all only applies to index, not alias
